docs.cask.co/www/resources/scripts/redirect-page.js

```diff
@@ -0,0 +1,44 @@
+/*
+ * redirect-page.js
+ * ----------------------------
+ *
+ * JavaScript for redirecting to an appropriate lower-level 404 page
+ *
+ * :copyright: © Copyright 2016 Cask Data, Inc.
+ * :license: Apache License, Version 2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ * 
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ * 
+ */
+
+(function() {
+  var products = new Array( "cdap", "coopr", "tigon" );
```

Comment:
The browser wouldn't be able to read the file, but we could generate this file (redirect-page.js) dynamically. I've added that to the `build.sh` script.